src/userApp/controllers.js

```diff
@@ -0,0 +1,64 @@
+/* c8 ignore next 80 */
+import { v4 as uuid } from 'uuid';
+import handleResponse from '../controllers/handleResponse.js';
+import { User } from '../models/user.js';
+import ResetToken from './models.js';
+import sendResetEmail from './utils.js';
+
+/* c8 ignore next 10 */
```

Comment:
these lines can be tested manually you don't need to ignore them

Reply:
thanks, Mr yvad.